x-pack/test_serverless/functional/test_suites/common/discover/group2/_adhoc_data_views.ts

```diff
@@ -45,7 +44,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await security.testUser.setRoles(['kibana_admin', 'test_logstash_reader']);
       await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover.json');
       await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
-
+      // TODO: Serverless tests require login first
+      await PageObjects.svlCommonPage.loginWithPrivilegedRole();
```

Comment:
For now we're try to have the serverless tests mirror the stateful tests as much as possible to make future deduplication efforts easier, but this could make sense to consider as part of those efforts.